core/src/mindustry/entities/comp/PayloadComp.java

```diff
@@ -61,7 +61,7 @@ void pickup(Building tile){
         tile.pickedUp();
         tile.tile.remove();
         payloads.add(new BuildPayload(tile));
-        Fx.unitPickup.at(tile);
+        Fx.breakBlock.at(tile, tile.block.size);
```

Comment:
Why did you change this? The block isn't being broken; this effect is misleading and doesn't look good.

Reply:
Because it's hard to see whether you picked it up or not. However with new ui it's more obvious.